docs/guides/trace/capabilities.md

```diff
@@ -78,7 +78,7 @@ did not use them:
 The meaning of the columns is:
 
 * `CAP`: capability number.
-* `NAME`: capability name in a human friendly format.
+* `CAPNAME`: capability name in a human friendly format.
 * `AUDIT`: whether the kernel should audit the security request or not.
 * `VERDICT`: whether the capability was present (allow) or not (deny)
```

Comment:
For another PR: We should add this information to the `capabilitiesTypes.Event` using the `columnDesc` tag.